### PR TITLE
added initial html and css pages

### PR DESCRIPTION
**Pull request by utleroy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        <section>
+          <p><img src="img/yodaGreen.png"></p>
+          <p class="justifyText">Yoda's voice recognition makes him the most realistic Star Wars toy ever! He speaks 115 phrases that lead to Jedi Wisdom and Lightsaber mastery.
+            Yoda's realistic movements display unmatched Jedi Lightsaber skills; watch Yoda perform his signature 360 Lightsaber attack!
+            Legendary Yoda is for Jedi's aged 6+. 6 C batteries required. Batteries not included.
+            Includes: 1 Legendary Yoda, 1 Lightsaber, 1 Cane
+          </p>
+          <p class="greybar">Availability</p>
+        </section>
+        <div class="text-pad">
```
Comment (copypastedeveloper) [inline index.html]: Can you do this without adding an extra div?  Lets keep all of the `<section/>`'s at the same level.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+body {
```
Comment (copypastedeveloper) [inline main.css:1]: This is basically saying, hey `<body>`, do what you would already be doing anyway.  It's not breaking anything, but it's not doing anything either.

**Comment by copypastedeveloper** [comment]
## Requirements:
- [x] Check out requirement #5 from the [Additional Style Requirements](https://github.com/nss-evening-cohort-04/mastery-exercise-html-utleroy#additional-style-requirements) section.  Make sure you are bolding the details, not the titles.
- [x] Fix [this](https://github.com/nss-evening-cohort-04/mastery-exercise-html-utleroy/pull/1#discussion_r73973234) in all of your cards.
- [x] Fix [this](https://github.com/nss-evening-cohort-04/mastery-exercise-html-utleroy/pull/1/files#r73973784) in your css.  not a big deal, but just not necessary.

Everything looks really nice.  Just a few nitpicks really.  Clean up and @ncgonz mention me and I'll give you a thumbs up.

**Comment by utleroy** [comment]
@ncgonz Changes corrected. I think...

**Comment by copypastedeveloper** [comment]
@utleroy no new commits?

**Comment by utleroy** [comment]
@ncgonz that would be my fault. Just pushed the changes.

**Comment by copypastedeveloper** [comment]
Its Happening!!!!

👍

**Pull request merged**